spec.html

```diff
@@ -19727,6 +19992,31 @@ <h1>Static Semantics: Early Errors</h1>
           It is a Syntax Error if PropName of |MethodDefinition| is `"prototype"`.
         </li>
       </ul>
+      <emu-grammar>
+        FieldDefinition :
+          ClassElementName Initializer?
+      </emu-grammar>
+      <ul>
+        <li>It is a Syntax Error if ContainsArguments of |Initializer| is *true*.</li>
+        <li>It is a Syntax Error if |Initializer| is present and |Initializer| Contains |SuperCall| is *true*.</li>
+      </ul>
+
+      <emu-grammar>
+        ClassBody : ClassElementList
+      </emu-grammar>
+      <ul>
+        <li>It is a Syntax Error if PrivateBoundIdentifiers of |ClassBody| contains any duplicate entries.</li>
+      </ul>
+
+      <emu-grammar>ClassElement : FieldDefinition `;`</emu-grammar>
+      <ul>
+        <li>It is a Syntax Error if PropName of |FieldDefinition| is `"constructor"`.</li>
```

Comment:
`class { static constructor() {} }` is allowed; does this imply that `class { static constructor; }` is disallowed?